envs/picard.yaml

```diff
@@ -0,0 +1,5 @@
+name: picard
```

Comment:
There is no `picard` in this environment. Is it a mistake?

Reply:
No, it's an env FOR picard, that's why it is called so

Reply:
So, where is the picard itself?

Reply:
It is downloaded inside rule
https://github.com/genxnetwork/grape/blob/69c6ffd2ae2a379bef7e56ed37b32df9847f006c/workflows/reference/Snakefile#L126-L138

Reply:
Let's rename to `java.yaml` since it's for java, not for `picard`. Btw, we should consider to add `picard` to the dependencies and do not download it each time we do something since it's quite weird.

Reply:
I renamed it to java.yaml. I think we can add picard + java to dependencies as another task.

Reply:
Yes! You can create suck task.